.github/workflows/error-ref-publisher.yaml

```
@@ -0,0 +1,64 @@
+name: Meshkit Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'layer5io/meshkit'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # token here with write access to meshkit repo
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```

```suggestion
        run: |
          go get github.com/layer5io/meshkit/cmd/errorutil
          go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```
meshkit hasn't been bumped yet. we'll need to make sure that we run the release with the latest changes to avoid the component_info.json bug.